boards/arm/cxd56xx/common/src/cxd56_gs2200m.c

```diff
@@ -47,6 +47,20 @@
 #define SPI_TX_MAXSIZE (CONFIG_CXD56_DMAC_SPI5_TX_MAXSIZE)
 #define SPI_RX_MAXSIZE (CONFIG_CXD56_DMAC_SPI5_RX_MAXSIZE)
 
+#if defined(CONFIG_WIFI_BOARD_IS110B_HARDWARE_VERSION_10B)
+/* v1.0b */
+#define GS2200M_GPIO_37          (PIN_UART2_CTS)
+#define GS2200M_EXT_RTC_RESET_IN (PIN_EMMC_DATA3)
+#elif defined(CONFIG_WIFI_BOARD_IS110B_HARDWARE_VERSION_10C)
+/* v1.0c */
+#define GS2200M_GPIO_37          (PIN_EMMC_DATA2)
+#define GS2200M_EXT_RTC_RESET_IN (PIN_EMMC_DATA3)
+#else
```

Comment:
(nitpick) is this actually possible to happen? how about using `elif` statement explicitly check the configuration?

Reply:
> (nitpick) is this actually possible to happen? how about using `elif` statement explicitly check the configuration?

`else` means default setting. In Kconfig, I configure v1.0A as default haredware version. So I not check CONFIG_WIFI_BOARD_IS110B_HARDWARE_VERSION_10A in `else` codes.

Reply:
> lgtm, btw could you make sure that there is no style warning with `tools/nxstyle`?

Yes. I have done it.
```bash
$ tools/nxstyle -v 1 boards/arm/cxd56xx/common/src/cxd56_gs2200m.c
/home/barry-xu/Work/incubator_nuttx/nuttx/boards/arm/cxd56xx/common/src/cxd56_gs2200m.c: PASSED nxstyle check
```